README.md

```diff
@@ -23,20 +24,19 @@ The subs card will expand to fill the width of its containing element, so you wi
 <div class="o-subs-card" data-o-component="o-subs-card">
 	<div class="o-subs-card__top">
 		<div class="o-subs-card__copy-title">Subscription title</div>
-		<img class="o-subs-card__copy-image" src='/demos/src/demo-placeholder.png' alt='photo of FT subscription options'>
 		<div class="o-subs-card__charge">
-			<span class="o-subs-card__charge__value">£XX.XX</span> per week
 			<button class="o-subs-card__select-button">Select</button>
+			<div class="o-subs-card__charge__value">£XX.XX per week</div>
```

Comment:
Was the element changed from a span to a div in order to have a display value of block?

Reply:
The `span` was emboldening the price in the sentence, and the `div` now wraps around the whole phrase (be it '££ per week' or 'Pay based on use', so I don't think a span is appropriate anymore.